pxteditor/editor.ts

```diff
@@ -54,8 +55,8 @@ namespace pxt.editor {
         tracing?: boolean;
 
         highContrast?: boolean;
-
-        home?: boolean;
+        hideExperimentalBanner?: boolean;
+        hideWindowsStoreBanner?: boolean;
```

Comment:
the default state should be no experimental banner, so this state should be reversed.